app.rb

```diff
@@ -27,6 +27,23 @@ class Chitter < Sinatra::Base
    erb :'messages/index'
   end
 
+  get '/messages/post' do 
+    erb :'messages/post'
```

Comment:
i like that the route and the filename are the same - makes things clear to the developer